app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UserSerializer
+  include JSONAPI::Serializer
```

Comment:
We can create an `ApplicationSerializer` file and put this into it. Then you can let this serializer inherits it